tests/multi_output_models.py

```diff
@@ -0,0 +1,53 @@
+from typing import List
+
+import numpy as np
+
+from sklearn.utils.multiclass import type_of_target
+
+from scikeras.utils.transformers import ClassifierLabelEncoder
+from scikeras.wrappers import KerasClassifier
+
+
+class MultiLabelTransformer(ClassifierLabelEncoder):
+    def fit(self, X: np.ndarray) -> "MultiLabelTransformer":
+        self._target_type = type_of_target(X)
+        if self._target_type != "multilabel-indicator":
```

Comment:
_We_ may only call it with `make_classification()`/`binary` targets, but these are used for the sklearn estimator checks, which call it with `binary` and `multiclass` as well. That's why it forwards those categories instead of raising an error.